<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>demoClass</class>
 <widget class="QDialog" name="demoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>886</width>
    <height>625</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>demo</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,0" columnstretch="0" rowminimumheight="0,0" columnminimumwidth="0">
   <property name="sizeConstraint">
    <enum>QLayout::SetMaximumSize</enum>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="hlayout_display" stretch="2,1">
     <item>
      <widget class="QLabel" name="img_label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="hlayout_button" stretch="0,0,0,0">
     <item>
      <widget class="QPushButton" name="selectImg_pushButton">
       <property name="text">
        <string>Select Image...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="fetch_pushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Fetch</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exit_pushButton">
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>selectImg_pushButton</tabstop>
  <tabstop>fetch_pushButton</tabstop>
  <tabstop>plainTextEdit</tabstop>
  <tabstop>exit_pushButton</tabstop>
 </tabstops>
 <resources>
  <include location="demo.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>exit_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>demoClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>537</x>
     <y>422</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>404</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
